docs/synapse/userguides/model_update_v2_176_0.rst

```diff
@@ -0,0 +1,27 @@
+.. _userguide_model_v2_176_0:
+
```

Comment:
I want to manually cut the old updates out of the 2.x changelog where they are nicely formatted to back fill here.